internal/generate/olm-catalog/package_manifest_test.go

```diff
@@ -61,3 +122,95 @@ const packageManifestExp = `channels:
 defaultChannel: stable
 packageName: memcached-operator
 `
+
+func Test_NewPackageManifest(t *testing.T) {
```

Comment:
This test isn't really testing anything, since all `newPackageManifest` does is set a few fields on a type with well-defined values. Please remove.